data_collection/gazette/spiders/ba/ba_teixeira_de_freitas.py

```diff
@@ -1,7 +1,8 @@
-from gazette.spiders.base.doem import DoemGazetteSpider
-
+from datetime import date 
+from gazette.spiders.base.doem import DoemGazetteSpider 
 
 class BaTeixeiraDeFreitasSpider(DoemGazetteSpider):
     TERRITORY_ID = "2931350"
     name = "ba_teixeira_de_freitas"
     state_city_url_part = "ba/teixeiradefreitas"
+    start_date = date(2014, 12, 31) 
```

Comment:
```suggestion
    start_date = date(2021, 3, 2) 
```

Achado em: https://doem.org.br/ba/teixeiradefreitas/diarios/2021/3